native/src/constants/buildConfig.ts

```diff
@@ -9,55 +9,55 @@ import { INTEGREAT_ASSETS, MALTE_ASSETS, ASCHAFFENBURG_ASSETS } from 'build-conf
 import { CommonBuildConfigType } from 'build-configs/BuildConfigType'
 import aschaffenburgLoadingImage from 'build-configs/aschaffenburg/assets/app-icon-inverted.svg'
 import aschaffenburgAppIcon from 'build-configs/aschaffenburg/assets/app-icon-round.svg'
+import directionsIcon from 'build-configs/common/assets/directions.svg'
+import informationIcon from 'build-configs/common/assets/information.svg'
+import languageIcon from 'build-configs/common/assets/language.svg'
+import offlineIcon from 'build-configs/common/assets/offline.svg'
+import searchIcon from 'build-configs/common/assets/search.svg'
 import integreatLoadingImage from 'build-configs/integreat/assets/app-icon-inverted.svg'
 import integreatAppIcon from 'build-configs/integreat/assets/app-icon-round.svg'
 import integreatCityNotCooperatingIcon from 'build-configs/integreat/assets/city-not-cooperating.svg'
-import integreatIntroEventsIcon from 'build-configs/integreat/assets/intro-slides/Events.svg'
-import integreatIntroLanguageIcon from 'build-configs/integreat/assets/intro-slides/Language.svg'
-import integreatIntroSearchIcon from 'build-configs/integreat/assets/intro-slides/Search.svg'
 import malteLoadingImage from 'build-configs/malte/assets/app-icon-circle.svg'
 import malteAppIcon from 'build-configs/malte/assets/app-icon-round.svg'
-import malteIntroEventsIcon from 'build-configs/malte/assets/intro-slides/Events.svg'
-import malteIntroLanguageIcon from 'build-configs/malte/assets/intro-slides/Language.svg'
-import malteIntroSearchIcon from 'build-configs/malte/assets/intro-slides/Search.svg'
 
 type AssetsType = {
   AppIcon: React.JSXElementConstructor<SvgProps>
   LoadingImage: React.JSXElementConstructor<SvgProps>
   CityNotCooperatingIcon?: React.JSXElementConstructor<SvgProps>
   intro?: {
-    Events: React.JSXElementConstructor<SvgProps>
+    directions: React.JSXElementConstructor<SvgProps>
     Language: React.JSXElementConstructor<SvgProps>
     Search: React.JSXElementConstructor<SvgProps>
+    Offline: React.JSXElementConstructor<SvgProps>
+    information: React.JSXElementConstructor<SvgProps>
   }
 }
 
 const buildConfig = (): CommonBuildConfigType => loadBuildConfig(name, COMMON)
 
 export const buildConfigAssets = (): AssetsType => {
   const assetsName = buildConfig().assets
+  const commonIntros = {
+    directions: directionsIcon,
+    Language: languageIcon,
+    Search: searchIcon,
+    Offline: offlineIcon,
+    information: informationIcon,
+  }
 
   if (assetsName === INTEGREAT_ASSETS) {
     return {
       AppIcon: integreatAppIcon,
       LoadingImage: integreatLoadingImage,
       CityNotCooperatingIcon: integreatCityNotCooperatingIcon,
-      intro: {
-        Events: integreatIntroEventsIcon,
-        Language: integreatIntroLanguageIcon,
-        Search: integreatIntroSearchIcon,
-      },
+      intro: commonIntros,
```

Comment:
Very nice that we can just use this for every build config, nice implementation! :tada:  We could even think of implementing this for aschaffenburg easily :thinking: I'll get back to you in regards to that.